trpcultivate_phenotypes/src/TripalCultivateValidator/ValidatorTraits/ColumnIndices.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Drupal\trpcultivate_phenotypes\TripalCultivateValidator\ValidatorTraits;
+
+/**
+ * Provides setters focused on configuring a validator to look at particular
+ * column within a row of a provided file
+ */
+trait ColumnIndices {
+
+  /**
+   * Sets the array of indices for which the validator to do its checks on.
+   *
+   * @param array $indices
+   *   An array of integers or keys which correspond to columns in a row of
+   *   delimited values that the validator instance should act on
```

Comment:
This description is still a bit inclear to me. What do you think of this?

```suggestion
   *   An array where each value is the key of the column the validator instance should
   *   act on. It must be either an integer or string. For more detail, see the
   *   definition of indices in the docblock above.
```